pkg/manifestgen/manifestgen.go

```diff
@@ -35,7 +38,9 @@ func defaultDepsolver(cacheDir string, packageSets map[string][]rpmmd.PackageSet
 	solver := dnfjson.NewSolver(d.ModulePlatformID(), d.Releasever(), arch, d.Name(), cacheDir)
 	depsolvedSets := make(map[string]dnfjson.DepsolveResult)
 	for name, pkgSet := range packageSets {
-		res, err := solver.Depsolve(pkgSet, sbom.StandardTypeNone)
+		// XXX: is there harm in always generating an sbom?
+		// (expect for slightly longer runtime?)
```

Comment:
I think there's not, but maybe @thozza knows more.

Reply:
None, except a slightly longer runtime and a bigger response from the `osbuild-depsolve-dnf`.

Reply:
I updated the comment.